src/main/scala/codacy/stylelint/DocGenerator.scala

```diff
@@ -48,14 +48,9 @@ object DocGenerator {
 
   def addNewPattern(patternName: String, default: Parameter.Value): Pattern.Specification = {
     val param = Set(Parameter.Specification(Parameter.Name(patternName), default))
-    val enabled = DefaultPatterns.patterns.contains(patternName)
-    Pattern.Specification(
-      Pattern.Id(patternName),
-      Result.Level.Err,
-      Pattern.Category.CodeStyle,
-      None,
-      param,
-      enabled = enabled)
+    val enabled = CodacyValues.patternsEnabled.contains(patternName)
+    val level = if (CodacyValues.possibleErrorsPatterns.contains(patternName)) Result.Level.Warn else Result.Level.Info
```

Comment:
Even the `possibleErrorsPatterns`, we prefer to have them as Warning on our side? I am fine with it, just double checking :D

Reply:
I just went by gut feel, while looking at the list of patterns they felt like they weren't Critical 😬 

@alerizzo Can you look into more detail in the list and share your opinion? https://stylelint.io/user-guide/rules/list/#possible-errors